pkg/validate/security_context_linux.go

```diff
@@ -236,9 +237,8 @@ var _ = framework.KubeDescribe("Security Context", func() {
 			}, time.Minute, time.Second*4).Should(Equal(runtimeapi.ContainerState_CONTAINER_RUNNING))
 
 			By("get nginx container pid")
-			command := []string{"cat", "/proc/1/cmdline"}
-			o := execSyncContainer(rc, containerID, command)
-			Expect(o).To(ContainSubstring("master process"))
+			command := []string{"sh", "-c", `while ! cat /proc/1/cmdline | grep "master process"; do sleep 1; done`}
```

Comment:
nit: this command will not produce any output so if test will fail due to timeout, it will be impossible to say what the content of the `/proc/1/cmdline` was. Would the similar command as in the comment above work better?

Reply:
One of the possible solutions might be using `Eventually` with `ExecSync` like my initial patch https://github.com/kubernetes-sigs/cri-tools/commit/3ef4d3eabde5a76540f73594d5cd466636d8b009 ?
@mikebrow WDYT?

Reply:
I think what he's saying is "while [ ! -f /proc/1/cmdline ]; do sleep 1; done", "&&", "cat", "/proc/1/cmdline"

Reply:
I don't think the command solves this problem because the timing-sensitive point here is the *contents* of the file rather than the existence.

After the entrypoint of the nginx image starts, `/proc/1/cmdline` exists but the initial content is `sh /docker-entrypoint.sh nginx -g daemon off;`. After a while, the entrypoint rewrites the content of this file to `nginx: master process nginx -g daemon off;` which the test expects to read. If the test reads this file before the rewrite occurs, the test fails. This is what I saw in our environment and this PR intends to fix it.

So we need to wait for the contents rather than the existence. (Maybe we should move to a more test-friendly image than nginx but let's do it on a following-up PR)

Reply:
But thanks for your suggestion!

Reply:
OK, let's get this merged.

@ktock would you like open an issue to track the following improvements?